src/services/lnd/health.ts

```diff
@@ -13,24 +26,23 @@ const isUpLoop = (param) => setInterval(() => isUp(param), refresh_time)
 
 export const isUp = async (param): Promise<void> => {
   let active
+  let state
   const { lnd, socket } = param
 
   try {
     // will throw if there is an error
-    await getWalletInfo({ lnd })
+    state = await getWalletStatus({ lnd })
     active = true
   } catch (err) {
     baseLogger.warn({ err }, `can't get wallet info from ${socket}`)
     active = false
   }
 
-  if (active && !param.active) {
+  if (state == "RPC_ACTIVE") {
     lndStatusEvent.emit("started", param)
-  }
-
-  if (!active && param.active) {
+  } else {
     lndStatusEvent.emit("stopped", param)
-  }
+	}
```

Comment:
Whitespace... I think you need to check the config of your editor to auto-run prettier on save.